build/scripts/contiv-vol-run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+
+# cleanup the older container
+docker rm -f "$1" &>/dev/null || :
+
+## test for shared mount capability
+if ! grep "MountFlags" /lib/systemd/system/docker.service | grep shared &>/dev/null
+then
+  echo 1>&2 "Docker cannot run volplugin in its current state."
+  echo 1>&2 "You must change docker's systemd MountFlags to equal 'shared'"
```

Comment:
just curious, do you know why is this required?

I have been trying out kubernetes bring up in containers recently (using hyperkube) and looks like it requires MountFlags to be set to slave (https://github.com/kubernetes/kubernetes/issues/4869#issuecomment-194149162). So, we are kind of conflicting with that requirement.